### PR TITLE
chore: bump oxc_resolver

### PR DESCRIPTION
**Pull request by h-a-n-a**
## Summary

closes https://github.com/web-infra-dev/rspack/issues/5151

## Require Documentation?

- [ ] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

**Comment by h-a-n-a** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #5245** <a href="https://app.graphite.dev/github/pr/web-infra-dev/rspack/5245?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
    * **PR #5152** <a href="https://app.graphite.dev/github/pr/web-infra-dev/rspack/5152?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/web-infra-dev/rspack/5245?utm_source=stack-comment).

File: crates/node_binding/src/lib.rs [hunk 1]
```diff
@@ -69,7 +69,7 @@ impl Rspack {
     output_filesystem: ThreadsafeNodeFS,
     js_loader_runner: JsFunction,
   ) -> Result<Self> {
-    Self::prepare_environment(&env);
+    Self::init_napi(&env);
```
Comment (Boshen) [inline crates/node_binding/src/lib.rs:72]: irrelevant change?

File: crates/rspack_error/src/diagnostic.rs [hunk 2]
```diff
@@ -97,6 +97,7 @@ impl Diagnostic {
       } else {
         GraphicalTheme::unicode_nocolor()
       })
+      .with_width(200)
```
Comment (Boshen) [inline crates/rspack_error/src/diagnostic.rs:100]: useless change?

File: packages/rspack/tests/StatsTestCases.test.ts [hunk 3]
```diff
@@ -80,9 +80,18 @@ describe("StatsTestCases", () => {
 				expect(statsJson.errors.length === 0);
 			}
 
+			statsJson.errors?.forEach(e => {
+				e.message = e.message.replace(/\n[ ]+│ /, "");
```
Comment (Boshen) [inline packages/rspack/tests/StatsTestCases.test.ts:84]: Extract this `replace` into a function and comment on why this is necessary?

**Review by Boshen — APPROVED** [review]

**Pull request merged**